notebooks/406-human-pose-estimation-3d/Visual3D/cvEngine.py

```diff
@@ -0,0 +1,103 @@
+# This is 3d Engine
+from object_3d import *
+from camera import *
+from projection import *
+from IPython.display import display, Image, clear_output
+import cv2
+
+class Engine3D:
+    """
+    Summary of class here.
+
+    This is an OpencV graphics interface to complete the graphics rendering pipeline.
+    
+    Attributes:
+        width, height:   Size of the view window.
+    """
+    
+    def __init__(self, width=500, height=450, cam=[3, 2, -6]):
+        # screen init
+        self.WIDTH, self.HEIGHT = width, height
+        self.H_WIDTH, self.H_HEIGHT = self.WIDTH // 2, self.HEIGHT // 2
+        self.FPS = 60
+        self.screen = np.zeros((self.HEIGHT, self.WIDTH, 3), dtype=np.int8)
+        self.Cam = cam
+        self.create_objects()
+        
+        # set color
+        self.Color = {'red' : (255, 0, 0), 
+                      'green' : (0, 255, 0),
+                      'blue' : (0, 0, 255)}
+        
+        # object
+        # self.world_axes = world_axes
```

Comment:
what's the use of this command line here? If not, please delete it. If uncomment it could be further used for another purpose, please leave some comments.